### PR TITLE
Fix snowcrash request capture groups

### PR DESCRIPTION
**Pull request by w-vi**

### CODE REVIEW

File: src/PayloadParser.h [hunk 1]
```diff
@@ -394,7 +394,7 @@ namespace snowcrash {
                                    const ParseResultRef<Payload>& out) {
 
             const char* regex;
-            mdp::ByteBuffer mediaType;
+	    mdp::ByteBuffer mediaType;
```
Comment (kylef) [inline src/PayloadParser.h]: The indentation here is missing a space.

Comment (w-vi) [inline src/PayloadParser.h]: Seems like a github bug, look at the code https://github.com/apiaryio/snowcrash/blob/kylef/request-syntax/src/PayloadParser.h#L397

Comment (kylef) [inline src/PayloadParser.h]: Actually, there is one tab and four spaces at the start of this line. Changing the tab back to spaces should resolve.

Comment (w-vi) [inline src/PayloadParser.h]: You are right, I should restart emacs once in a while, the automatic formatting on save got messed up somehow.

**Pull request merged**